tests/golos.ctrl_tests.cpp

```diff
@@ -180,24 +181,28 @@ BOOST_FIXTURE_TEST_CASE(create_community, golos_ctrl_tester) try {
     BOOST_CHECK_EQUAL(err.no_symbol, ctrl.detach_acc(_carol));
 
     BOOST_TEST_MESSAGE("--- test fail when create community with bad parameters");
-    BOOST_CHECK_EQUAL(err.no_msig_acc, ctrl.set_params(ctrl.default_params(N(nobody), _token)));
+    BOOST_CHECK_EQUAL(err.no_msig_acc, ctrl.set_params(ctrl.default_params(N(nobody), _token, cfg::vesting_name)));
     // BOOST_CHECK_EQUAL(err.max_witness0, ctrl.set_params(ctrl.default_params(BLOG, symbol(-1,"bad"))));   // is it handled by serializer?
-    BOOST_CHECK_EQUAL(err.max_witness0, ctrl.set_params(ctrl.default_params(BLOG, _token, 0)));
-    BOOST_CHECK_EQUAL(err.max_wit_votes0, ctrl.set_params(ctrl.default_params(BLOG, _token, 21, 0)));
+    BOOST_CHECK_EQUAL(err.max_witness0, ctrl.set_params(ctrl.default_params(BLOG, _token, cfg::vesting_name, 0)));
+    BOOST_CHECK_EQUAL(err.max_wit_votes0, ctrl.set_params(ctrl.default_params(BLOG, _token, cfg::vesting_name, 21, 0)));
 
     BOOST_TEST_MESSAGE("------- permission thresholds");
-    BOOST_CHECK_EQUAL(err.smaj_lt_maj, ctrl.set_params(ctrl.default_params(BLOG, _token, 21, 30, _update_auth_period, 3,5,0)));
-    BOOST_CHECK_EQUAL(err.smaj_lt_min, ctrl.set_params(ctrl.default_params(BLOG, _token, 21, 30, _update_auth_period, 3,0,5)));
-    BOOST_CHECK_EQUAL(err.maj_lt_min,  ctrl.set_params(ctrl.default_params(BLOG, _token, 21, 30, _update_auth_period, 0,2,5)));
-
-    BOOST_CHECK_EQUAL(err.smaj_gt_max, ctrl.set_params(ctrl.default_params(BLOG, _token, 21, 30, _update_auth_period, 22,2,1)));
-    BOOST_CHECK_EQUAL(err.maj_gt_max,  ctrl.set_params(ctrl.default_params(BLOG, _token, 21, 30, _update_auth_period, 0,22,1)));
-    BOOST_CHECK_EQUAL(err.min_gt_max,  ctrl.set_params(ctrl.default_params(BLOG, _token, 21, 30, _update_auth_period, 0,0,22)));
-    BOOST_CHECK_EQUAL(err.maj_gt_smaj, ctrl.set_params(ctrl.default_params(BLOG, _token, 21, 30, _update_auth_period, 2,0,0)));
-    BOOST_CHECK_EQUAL(err.min_gt_smaj, ctrl.set_params(ctrl.default_params(BLOG, _token, 21, 30, _update_auth_period, 2,1,0)));
-    BOOST_CHECK_EQUAL(err.min_gt_maj,  ctrl.set_params(ctrl.default_params(BLOG, _token, 21, 30, _update_auth_period, 15,1,0)));
+    BOOST_CHECK_EQUAL(err.smaj_lt_maj, ctrl.set_params(ctrl.default_params(BLOG, _token, cfg::vesting_name, 21, 30, _update_auth_period, 3,5,0)));
```

Comment:
please move vesting_acc parameter to last and assign default value to it, so you won't need to pass `cfg::vesting_name` every time (all this lines will became same as before)